main/src/main/java/cgeo/geocaching/CachePopupFragment.java

```diff
@@ -271,6 +277,28 @@ public void onClick(final View arg0) {
         }
     }
 
+
+    private class MoveCacheClickListener implements View.OnLongClickListener {
+
+        @Override
+        public boolean onLongClick(final View v) {
+            if (progress.isShowing()) {
+                showToast(res.getString(R.string.err_detail_still_working));
+                return false;
+            }
+
+            new StoredList.UserInterface(getActivity()).promptForListSelection(R.string.cache_menu_move_list,
+                    this::moveCacheToList, true, -1);
+
+            return true;
+        }
+
+        private void moveCacheToList(final Integer listId) {
```

Comment:
Since this is a "move", shouldn't this method also remove the case from the source list?

Reply:
In the called method DataStore.saveLists this is done, first remove from all lists and add it to the given lists, in this case only a single one.